packages/components/src/components/vertical-tab/vertical-tab-wrapper.jsx

```diff
@@ -7,4 +7,4 @@ const VerticalTabWrapper = ({ children, className }) => (
     </div>
 );
 
-export { VerticalTabWrapper };
+export default VerticalTabWrapper;
```

Comment:
Please prefix all classnames in this file with `dc-` (with updates to their respective stylesheet).